docs/src/components/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -54,6 +56,14 @@ const PlatformNav = ({
     autoScrollToSelected
     className={getClassName('bpkdocs-page-wrapper__platform-switcher')}
   >
+    <BpkHorizontalNavItem
+      name="design"
+      disabled={disabledDesignTab}
```

Comment:
It's `disable`, not `disabled` on the other platforms

```suggestion
      disabled={disableDesignTab}
```

Reply:
Good spot all fixed now